<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  
  <param name="yolov5_path"       value="$(find yolov5_findball)/yolov5"/>
  <param name="use_cpu"           value="true" />

  <!-- red -->
  <node pkg="yolov5_findball" type="decectball2.py" name="yolov5_findball" output="screen" >
 
    <param name="weight_path"       value="$(find yolov5_findball)/weights/bestred532.pt"/>
    <param name="image_topic"       value="/camera/image_raw2" />
    <param name="pub_topic"         value="/camera1/ball_position" />
    <param name="camera_frame"       value="camera_color_frame"/>
    <param name="conf"              value="0.3" />
   
  </node>
  <!-- blue -->
<!-- 
  <node pkg="yolov5_findball" type="decectball2.py" name="yolov5_findball" output="screen" >
 
    <param name="weight_path"       value="$(find yolov5_findball)/weights/bestblue83.pt"/>
     <param name="image_topic"       value="/camera/image_raw2" />
    <param name="pub_topic"         value="/camera1/ball_position" />
    <param name="camera_frame"       value="camera_color_frame"/>
    <param name="conf"              value="0.3" />
   
  </node> -->

  <!-- <node name="camera_ros" pkg="yolov5_ros" type="camera.py" output="screen"> -->
  <node pkg="yolov5_findball" type="camera.py" name="camera_findball" output="screen" >
   <param name="dev"   value="2"/>
  </node>


</launch> 

